assets/src/stories-editor/components/block-navigation/index.js

```diff
@@ -70,7 +70,15 @@ function BlockNavigation( { blocks, selectBlock, selectedBlockClientId } ) {
 					selectBlock={ selectBlock }
 				/>
 			) }
-			{ ! hasBlocks && (
+			{ /* Add CTA block as a separate item to exclude it from DropZone. */ }
+			{ callToActionBlock && (
+				<BlockNavigationItem
```

Comment:
So while the other blocks are in a nice `ul` list, this item here is just a simple `div`? Seems a bit odd from a semantics point of view.

Reply:
Maybe, however, the items in the list are draggable and within dropzone, so it doesn't feel like it should be in the same list either -- it behaves very differently. 🤷‍♀Do you think it would be better to create a separate list for that?

Reply:
Or are you thinking that it should still be in the same list, just behave differently?

Reply:
> Or are you thinking that it should still be in the same list, just behave differently?

I was thinking this (with the necessary logic being in `BlockNavigationList`), however just having a separate list might be more feasible.

Reply:
Alright, will move it to `BlockNavigationList` and add a separate list.

Reply:
Changes made within 0b14eea.

Note that there is some room left between the lists to separate the draggable items from the static CTA link, seems to make sense when testing, however, let me know if you have objections.